packages/sdk-scripts/src/lib/Context.ts

```diff
@@ -53,15 +56,30 @@ export class Context {
       toString: () => sdkIndexExports,
     };
 
+    const contractDoc = {
+      name: `providers-of-${changeCase.paramCase(this.serviceSeg)}`,
+      description: this.namespaceI18nMap.get("description"),
+      providers: this.providerContracts,
+    };
+
     return [
       ...Array.from(this.apiMap.values()),
       ...Array.from(this.modelMap.values()),
       ...apiByModelIndexes,
       ...modelByServiceIndexes,
       sdkIndex,
-    ].map<FileWithContent>((file) => [
-      path.join(sdkRoot, "src", file.filePath + ".ts"),
-      prettier.format(file.toString(), { parser: "typescript" }),
-    ]);
+    ]
+      .map<FileWithContent>((file) => [
+        path.join(sdkRoot, "src", file.filePath + ".ts"),
+        prettier.format(file.toString(), { parser: "typescript" }),
+      ])
+      .concat([
+        [
+          path.join(sdkRoot, "contract.json"),
+          prettier.format(JSON.stringify(contractDoc, null, 2), {
+            parser: "json",
+          }),
```

Comment:
`JSON.stringify` 已经美化了，不需要 prettier 了：

```suggestion
          JSON.stringify(contractDoc, null, 2),
```